### PR TITLE
Update lpc55 signing

### PR DESCRIPTION
**Pull request by labbott**
The lpc55_support crate got refactored and also dropped the OpenSSL
dependency. Update the paths accordingly.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -66,8 +66,6 @@ jobs:
             deps: |
               Invoke-WebRequest -Uri https://github.com/steveklabnik/arm-none-eabi-objcopy/releases/download/9-2020-q2-update/arm-none-eabi-objcopy.exe -OutFile "$Env:RUNNER_TEMP\arm-none-eabi-objcopy.exe"
               echo $Env:RUNNER_TEMP >> $Env:GITHUB_PATH
-              echo "VCPKG_ROOT=$Env:VCPKG_INSTALLATION_ROOT" >> $Env:GITHUB_ENV
```
Comment (cbiffle) [inline .github/workflows/build.yml:69]: Merge conflict?

**Review by cbiffle — CHANGES_REQUESTED** [review]

**Review by cbiffle — COMMENTED** [review]
Not a merge conflict, evidently, let's see if this rescinds my -1 review

**Comment by cbiffle** [comment]
> Not a merge conflict, evidently, let's see if this rescinds my -1 review

Apparently not.

**Review by steveklabnik — APPROVED** [review]
I can barely believe it, but a 

```console
$ vcpkg remove openssl:x64-windows --recursive
$ cargo clean
$ cargo xtask dist .\app\lpc55xpresso\app.toml
```

successfully builds!

This also fixes #298.

**Pull request merged**